### PR TITLE
Fix truncated log with Git Extensions

### LINKED ISSUE #8580: Log truncated when "git clean"
## Current behaviour

Open Commands|Clean working directory
Press "Preview"

The showmodal window says :

image

but the log in the "Clean working directory" windows is truncated.

image

There is also some no printable characters.

## Expected behaviour

No truncate.

## Did this work in previous version of GitExtensions

No idea

## Environment

- Git Extensions 3.4.3.9999
- Build d4b0f48bbf3e39a71f5d7ff5231be5678d4aa0f1
- Git 2.28.0.windows.1
- Microsoft Windows NT 10.0.19041.0
- .NET Framework 4.8.4250.0
- DPI 96dpi (no scaling)

### PR DESCRIPTION
**Pull request by bansan85**
By using "-new_console:L:" instead of "-AnsiLog",
there is no more color information and date in log file.

Fix gitextensions/gitextensions#8580

### CODE REVIEW

**Comment by gerhardol** [comment]
https://conemu.github.io/en/NewConsole.html#example-8
I do not find -AnsiLog in the documentation, but -new_console seem OK

File: ConEmuWinForms/ConEmuSession.cs [hunk 1]
```diff
@@ -501,6 +495,12 @@ private static unsafe CommandLineBuilder Init_MakeConEmuCommandLine([NotNull] Co
 			}
 			cmdl.AppendSwitchIfNotNull("-cur_console:", $"{(startinfo.IsElevated ? "a" : "")}{sConsoleExitMode}");
 
+			// ANSI Log file
+			if (ansilog != null)
+			{
+				cmdl.AppendSwitchIfNotNull("-new_console:L:", ansilog.Directory.FullName);
```
Comment (gerhardol) [inline ConEmuWinForms/ConEmuSession.cs]: new_console is working, but I believe that cur_console should be used
```suggestion
				cmdl.AppendSwitchIfNotNull("-cur_console:L:", ansilog.Directory.FullName);

```

**Review by gerhardol — APPROVED** [review]
Functionality change seem to be OK
This affects all commands

File: ConEmuWinForms/ConEmuSession.cs (hunk 1 above)
Comment (bansan85) [reply ConEmuWinForms/ConEmuSession.cs]: I just tried, both works. I have no idea if it's new_console or cur_console.

**Review by gerhardol — APPROVED** [review]

**Comment by gerhardol** [comment]
@RussKie 
I plan to merge this before my PR (so in 23h)

**Pull request merged**